### PR TITLE
got people data

### PR DESCRIPTION
**Pull request by dzheng256**
2nd commit just formats unicode strings into regular ones. @RitwikGupta

### CODE REVIEW

**Comment by dzheng256** [comment]
added additional parameter to function to limit number of people returned, because it can take a while.

**Comment by dzheng256** [comment]
@RitwikGupta so I think the main problem with this over find.pitt.edu other than the fact that it has just less info overall is that if you're a student employee, your "unit" or department is replaced by the one you work for.

File: pittAPI.py [hunk 1]
```diff
@@ -375,29 +375,49 @@ class PeopleAPI:
...
+                response_str = response_str.replace('\u0026', '&')
+                response_str = response_str.replace('\\', '')
+                if '&start=' not in response_str:
+                    results.append("https://136.142.34.69" + response_str)
+
+                response = response[response.index('}') :]
+            for url in results:
+                #results is url
+                personurl = str(''.join(url))
+                f = urlopen(personurl)
```
Comment (RitwikGupta) [inline pittAPI.py]: This will result in an ssl CertificateError. Add a `ssl._create_unverified_context()` to this urlopen call.

File: pittAPI.py [hunk 2]
```diff
@@ -375,29 +375,49 @@ class PeopleAPI:
     def __init__(self):
         pass
 
-    def get_person(self, query):
+    def get_person(self, query, maxPeople):
```
Comment (RitwikGupta) [inline pittAPI.py]: Make `maxPeople` an optional requirement, it's currently required.

File: pittAPI.py [hunk 3]
```diff
@@ -375,29 +375,49 @@ class PeopleAPI:
...
+            url = "https://136.142.34.69/people/search?search=Search&filter={}&_region=kgoui_Rcontent_I0_Rcontent_I0_Ritems"
+            url += '&_region_index_offset=' + str(tens) + '&feed=directory&start=' + str(tens)
+            cmd = 'curl -k -s ' + '"'  + url + '"'
+
+
+            cmd = cmd.format(query)
+            response = subprocess.check_output(cmd, shell=True)
+            if not response: #no more responses, so break
+                break
+            #print(response)
```
Comment (RitwikGupta) [inline pittAPI.py]: Remove

Comment (dzheng256) [inline pittAPI.py]: Wait, remove the comment? The if + break statement would be useful if the user gives a max of  50 but there's only like 2 results

**Comment by RitwikGupta** [comment]
No, revert that last commit. All you need is `urlopen(url, context=ssl._create_unverified_context())`

**Comment by dzheng256** [comment]
that didnt work, ssl doesn't always have _create_unverified_context()

**Comment by RitwikGupta** [comment]
Closes #10

**Pull request merged**